src/ensembl/src/shared/components/communication-framework/contact-us/contact-us-form/submitForm.ts

```diff
@@ -18,9 +18,34 @@ type FormFieldType = string | File | File[] | null;
 
 type Form = Record<string, FormFieldType>;
 
-export const submitForm = (form: Form) => {
+type TokenSuccessResponse = {
+  token: string;
+};
+
+export const submitForm = async (form: Form) => {
   const formData = buildFormData(form);
-  console.log('formData', formData); // eslint-disable-line no-console
+  const url = '/api/support/ticket';
```

Comment:
Shouldn't we specify the base URL `/api/support` as a constant to reuse it?

Reply:
🤷‍♂️ We could, but at the moment, this api is used only in two functions, and both these functions are in the same file. I don't know if extracting `/api/support` into a constant is going to help much.